lints.toml

```diff
@@ -70,4 +69,5 @@ allow = [
     # `assert!(!foo(bar))` is misread the majority of the time, while `assert_eq!(foo(bar), false)` is crystal clear
     'clippy::bool-assert-comparison',
     'clippy::blocks_in_conditions',
+    'unused_imports', # TODO: DENY THIS
```

Comment:
Why move this?

Reply:
Whoops, that was a holdover from earlier testing. I'll undo it.